### DISCUSSION TITLE
What should happen with the unused 3.1.38 milestone?

### DISCUSSION THREAD

**Discussion by EliahKagan** [gitpython-developers/GitPython#1750] [Q&A] [answered]
As I understand it, since #1706/#1707, milestones are no longer used as a regular part of GitPython releases. However, I noticed that the old [3.1.38 milestone](https://github.com/gitpython-developers/GitPython/milestone/68) is [still open](https://github.com/gitpython-developers/GitPython/milestones?state=open), even though that version has been released (and surpassed). Should it be closed? Deleted?

I suppose this should be an issue, though it's extremely minor and doesn't affect GitPython itself. I'm opening it as a discussion for continuity with #1706, which I had opened that way.

**Comment by Byron** [comment, marked as answer]
Thanks for the hint! It was forgotten, and is now closed.
